src/viz/Ensight_Stream.hh

```diff
@@ -52,8 +55,14 @@ private:
 
   // DATA
 
-  // The actual file stream.
-  std::ofstream d_stream;
+  // An optional parallel stream for domain decomposed geometry
```

Comment:
Can you change the comments to `//!` so doxygen can see them?

Reply:
Done.